src/components/GfcrCallout/GfcrCallout.jsx

```
@@ -56,7 +57,13 @@ const StyledParagraph = styled('p')`
   max-width: ${theme.spacing.maxTextWidth};
 `
 
-const DisableIndicatorsModal = ({ isOpen = false, disableGfcr, onDismiss }) => {
+const DisableIndicatorsModal = ({
+  isOpen = false,
+  disableGfcr,
+  onDismiss,
+  mermaidExploreLink,
```

ESLint

[error] 'mermaidExploreLink' is missing in props validation

(react/prop-types)